usaspending_api/broker/management/commands/fix_locations.py

```diff
@@ -0,0 +1,149 @@
+"""
+Idempotently re-runs location creation / matching code from broker FPDS data
```

Comment:
@catherinedevlin is a recipient fix in the works or is only place of performance an issue at the moment?

Reply:
I can add the recipient fixer pretty quickly.  I didn't know if we wanted to triage that out for the sake of runtime, though since the whole thing only took about 7 hours on dev anyway, there's probably no need to.  I'll add that now.